shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -28,7 +28,7 @@
     <ul class="menubar">
       {{#each items}}
         {{#unless onlyPopup}}
-          <li class="{{name}}">
+          <li {{inlineStyle}} class="{{name}}">
```

Comment:
Drew, your proposal results in style="" in the no-inline-style case, but
I'm OK with that, so I've made this change locally. (Will push for review
again shortly.)